packages/eslint-plugin/package.json

```diff
@@ -13,27 +13,27 @@
     "test": "yarn jest --passWithNoTests"
   },
   "dependencies": {
-    "@rnx-kit/eslint-plugin": "^0.2.5",
-    "@typescript-eslint/eslint-plugin": "^4.22.0",
-    "@typescript-eslint/experimental-utils": "^4.22.0",
-    "@typescript-eslint/parser": "^4.22.0",
-    "eslint-config-airbnb": "^18.2.1",
+    "@rnx-kit/eslint-plugin": "^0.2.13",
+    "@typescript-eslint/eslint-plugin": "^5.31.0",
+    "@typescript-eslint/experimental-utils": "^5.31.0",
+    "@typescript-eslint/parser": "^5.31.0",
+    "eslint-config-airbnb": "^19.0.4",
     "eslint-config-prettier": "^8.3.0",
     "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-plugin-deprecation": "^1.2.1",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^23.13.2",
-    "eslint-plugin-jsdoc": "^36.0.7",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-react": "^7.24.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-deprecation": "^1.3.2",
+    "eslint-plugin-import": "^2.25.0",
+    "eslint-plugin-jest": "^25.1.0",
+    "eslint-plugin-jsdoc": "^37.9.7",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.28.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
     "fs-extra": "^8.1.0",
     "minimatch": "^3.0.4",
     "@nrwl/tao": "13.8.1",
     "jju": "^1.4.0"
   },
   "peerDependencies": {
-    "eslint": "^7.0.0",
+    "eslint": "^7.32.0",
```

Comment:
not sure if this is necessary. Are there any eslint features starting with version 7.32.0 that our plugins use ? if not I'd leave it as is